src/components/pages/Resetpassword.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable max-len */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { Redirect, withRouter } from 'react-router-dom';
+import { ChangePasswordAction } from '../../actions/resetPasswordAction';
+import Input from '../global/Input';
+import Button from '../global/Button';
+
+export class ResetPassword extends React.Component {
+    state = {
+      newPassword: '',
+      confirmPassword: '',
+      isLoading: false,
+    };
+
+    // eslint-disable-next-line camelcase
+    UNSAFE_componentWillReceiveProps() {
```

Comment:
You do not this lifecycle method, you can set the isLoading from the handleSubmit